src/subscript/eclcompress/eclcompress.py

```diff
@@ -190,20 +188,22 @@ def acceptedvalue(valuestring: str) -> bool:
         return False
 
 
-def compress_multiple_keywordsets(keywordsets: list, filelines: list) -> list:
+def compress_multiple_keywordsets(
+    keywordsets: List[Tuple[int, int]], filelines: List[str]
+) -> List[str]:
     """Apply Eclipse type compression to data in filelines
 
     The list of strings given as input (filelines) is indexed
     by the tuples in keywordsets.
 
     Args:
-        keywordsets (list): List of 2-tuples, (start, end) indices in
+        keywordsets: List of 2-tuples, (start, end) indices in
             line number in the deck, referring to individual sections
             of distinct keywords.
-        filelines (list): list of lines (strings) from Eclipse deck, cleaned.
+        filelines: list of lines from Eclipse deck, cleaned.
 
     Returns:
-        list:  List of strings to be used as a replacement Eclipse deck
+        Strings to be used as a replacement Eclipse deck
```

Comment:
In another method you have switched to e.g. `List of strings [...]` and `List of 2-tuples [...]` , so maybe change this for consistency, even though the type hinting is even more descriptive?
```suggestion
        List of strings to be used as a replacement Eclipse deck
```

Reply:
The intention was not to say "list of something" when the type hinting already says the same (and it is included next to the text in sphinx-generated docs), only a textual description, and then "strings" in plural is the textual description of "List[str]"

Reply:
Consistency now attempted.